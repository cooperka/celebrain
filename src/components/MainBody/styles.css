.MainBody {
  display: flex;
  flex-direction: column;
}
