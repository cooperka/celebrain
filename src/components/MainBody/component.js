// @flow

import React from 'react';

import NavButton from '../NavButton';

import './styles.css';

function MainBody() {
  return (
    <div className="MainBody">
      <div className="title">Lovable. Simple. Refined.</div>
      <div className="subtitle">
        I listen to users and build the <i>minimal&nbsp;lovable&nbsp;solution</i> to their needs.
      </div>
      <div className="buttons">
        <NavButton bordered href="mailto:kevin@lsrfocus.com">Contact</NavButton>
      </div>
    </div>
  );
}

export default MainBody;
