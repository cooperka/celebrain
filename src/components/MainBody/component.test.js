// @flow

import React from 'react';
import renderer from 'react-test-renderer';

import MainBody from './component';

jest.mock('./Hero/component', () => 'Hero');
jest.mock('./Services/component', () => 'Services');

it('renders', () => {
  const tree = renderer.create(<MainBody />);
  expect(tree.toJSON()).toMatchSnapshot();
});
