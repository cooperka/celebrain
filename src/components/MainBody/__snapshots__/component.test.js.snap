// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="MainBody"
>
  <div
    className="title"
  >
    Lovable. Simple. Refined.
  </div>
  <div
    className="subtitle"
  >
    I listen to users and build the 
    <i>
      minimal lovable solution
    </i>
     to their needs.
  </div>
  <div
    className="buttons"
  >
    <div
      className="NavButton bordered"
    >
      <a
        className="material waves-effect waves-teal btn-flat"
        href="mailto:kevin@lsrfocus.com"
      >
        <div>
          Contact
        </div>
      </a>
    </div>
  </div>
</div>
`;
